src/ralph/lib/custom_fields/admin.py

```diff
@@ -135,3 +135,33 @@ def changeform_view(
         return super().changeform_view(
             request, object_id, form_url, extra_context
         )
+
+    def _create_formsets(self, request, obj, change):
+        """Helper function to generate formsets for add/change_view."""
```

Comment:
Please add comment that this is 99% not altered internal django method + comment around lines that are altered and why.

Reply:
Done